backend-tests/run

```diff
@@ -261,9 +261,9 @@ for suite in "${TEST_SUITES[@]}"; do
     fi
 
     if [ $run_tests_retcode -ne 0 ]; then
-        tmppath=$(mktemp ${HERE}/acceptance.XXXXXX)
+        tmppath=$(mktemp -u --tmpdir=. acceptance.XXXXXX)
```

Comment:
According to the man page:
```
       -u, --dry-run
              do not create anything; merely print a name (unsafe)
```

Reply:
it is unsafe in a sense that it can lead to races, and other not secure creation of files. but this does not apply. if you insist we can create the file here, and overwrite it later.

Reply:
I removed the flag, but it really did not matter here.